components/dropdown/dropdown.tsx

```diff
@@ -153,16 +159,19 @@ export default class Dropdown extends Component {
     }
 
     render() {
-        const { trigger, rtl, autoClose } = this.props;
+        const { rtl, autoClose } = this.props;
+        const trigger = this.props.trigger as React.ReactPortal;
 
-        const child = Children.only(this.props.children);
-        let content = child;
+        const child = Children.only(this.props.children) as React.ReactPortal;
+        let content = child as React.ReactChild;
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-expect-error
         if (typeof child.type === 'function' && child.type.isNextMenu) {
```

Comment:
Use "as" instead of ignore comment

Reply:
child.type as unknown as ReactElementNextMenuType).isNextMenu